tests/docker/globus.ubuntu20.Dockerfile

```diff
@@ -51,11 +55,16 @@ RUN mkdir /iRODS_DSI
 RUN chmod 777 /iRODS_DSI
 
 #### Get and install iRODS repo ####
-RUN wget -qO - https://packages.irods.org/irods-signing-key.asc | sudo apt-key add -
-RUN echo "deb [arch=amd64] https://packages.irods.org/apt/ $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/renci-irods.list
-RUN apt-get update
+#RUN wget -qO - https://packages.irods.org/irods-signing-key.asc | sudo apt-key add -
+#RUN echo "deb [arch=amd64] https://packages.irods.org/apt/ $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/renci-irods.list
+#RUN apt-get update
```

Comment:
Why comment these out?

Reply:
This was a duplicate and will be removed.